api/Document.json

```diff
@@ -8173,12 +8167,10 @@
               "notes": "This event handler attribute is not supported; however, the event itself is supported since IE 10. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."
             },
             "opera": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
+              "version_added": "12.1"
```

Comment:
If the Chrome data is correct, this should be removed in 15, and then added back in 49.

It might be worth re-testing some Opera versions though, I wonder if this shouldn't be a ranged version and this was added earlier than 12.1.

Reply:
I've checked this out in Opera Presto. I confirmed with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8816 that support for the event *was* added in 12.1 (event logged in 12.1, but not in 11.6). But the event handler attribute was never supported in Opera Presto. So notes will be needed here too.